### DISCUSSION TITLE
Are files gone for ever if discard changes has done it's work?

### DISCUSSION THREAD

**Discussion by Antistoffer** [community/community#29874] [Other Feature Feedback, Questions, & Ideas]
I fucked up.

And just deleted everything I have ever made in vs code and I just wanted to make sure there really isn't anything I can do to get my files back

I know the message says expressly states that the action can't be undone

**Comment by radar-blip** [comment]
Check here: https://support.github.com/request

Near bottom there is "restore deleted data" , might help.

**Reply by Antistoffer** [reply]
Okay I will try that thank you

**Reply by airtower-luna** [reply]
That helps only if the mistake was one that deleted stuff *on GitHub*, like a wrong force-push. They won't be able to restore local files for you.

**Reply by Antistoffer** [reply]
Damn okay thank you anyway guess I have to recreate some things :)
